jedi_remote.py

```diff
@@ -0,0 +1,127 @@
+from __future__ import print_function
+
+import json
+import os
+
+import jedi.api
+
+from subprocess import Popen, PIPE
+
+
+class JediRemote(object):
+    '''Jedi remote process communication client
+
+    This class provide jedi compatible API.
+    '''
+    python = 'python'
```

Comment:
Can this be made configurable in Vim?
May `'use_rpc'` could be changed to not use this RPC method when empty, but would otherwise be the path / command to use?